README.md

```diff
@@ -0,0 +1,60 @@
+# kubeadm-systemd
+
+## Introduction
+
+__kubeadm-systemd__ is a tool for creating a multi-node Kubernetes cluster
+on the single machine, created mostly for developers __of__ Kubernetes.
+
+It aims to be as similar to the solutions recommendend for the production
+clusters as possible.
+
+## Architecture
+
+![Architecture Diagram](architecture.png?raw=true "Architecture")
+
+kubeadm-systemd uses the following third-party components to
+achieve its goal:
+
+### CNI
+
+Raw systemd-nspawn needs systemd-networkd for automanaging the networking
+for containers, which is bad for developers trying to use nspawn on their
```

Comment:
networkd can do what you need but there is no real user interface for it. I'd say something like this...

We chose to use CNI to provide networking for systemd-nspawn as it's what is used in kubernetes itself and system-networkd, is currently trick to use.